src/content/overview/architecture/_index.md

```diff
@@ -2,6 +2,10 @@
 title: Architecture
 description: Technical overview of the Giant Swarm platform components and how a customer's developer platform can use them successfully.
 weight: 20
+menu:
```

Comment:
These changes don't seem to belong in this PR

Reply:
yeah let me pull from main